recipe/meta.yaml

```diff
@@ -182,15 +162,12 @@ outputs:
         - scipy
       source_files:
         - tests/
-        - conda/faiss/run_test.py
-        - conda/faiss-gpu/run_test.py
       imports:
         - faiss
       commands:
-        - python conda/faiss/run_test.py
-        # running the following test requires an actual GPU device, which is not available in CI
-        # - python conda/faiss-gpu/run_test.py  # [cuda_compiler_version != "None"]
         - python -m unittest discover tests
+        # running the following test requires an actual GPU device, which is not available in CI
+        # - python -m unittest discover faiss/gpu/test/
 
   # for compatibility with (& ease of migration from) existing packages in the pytorch channel
   - name: faiss-cpu
```

Comment:
Do these package have to have custom build strings?
It would normally suffice to make them:
```yaml
  - name: faiss-cpu
    build:
      skip: true  # [cuda_compiler_version != "None"]
    requirements:
      run:
        - faiss ={{ version }}=*_cpu
    test:
      commands:
        - exit 0

  - name: faiss-gpu
    build:
      skip: true  # [cuda_compiler_version == "None"]
    requirements:
      run:
        - faiss ={{ version }}=*_cuda
    test:
      commands:
        - exit 0
```

Reply:
Will try